e2e/pages/login/loginPage.js

```diff
@@ -14,5 +14,6 @@ module.exports = {
     I.fillField(this.fields.username, username);
     I.fillField(this.fields.password, password);
     I.click(this.submitButton);
+    I.wait(3);
```

Comment:
Similar wait is already added to `logInAndCreateCase`. If we add wait here we should remove it from actor.

Also we should use `waitForNavigation`, this fixed wait is very fragile.